space.fips.Fips.json

```diff
@@ -0,0 +1,42 @@
+{
+	"app-id": "space.fips.Fips",
+	"runtime": "org.kde.Platform",
+	"runtime-version": "5.9",
+	"sdk": "org.kde.Sdk",
+	"command": "fips",
+	"rename-icon": "fips",
+	"rename-desktop-file": "fips.desktop",
+	"finish-args": [
+		"--share=ipc",
+		"--share=network",
+		"--socket=x11",
+		"--socket=wayland",
+		"--filesystem=home:ro",
+		"--device=dri"
+	],
+	"modules": [
+		{
+			"name": "fips",
+			"buildsystem": "cmake",
+			"config-opts": [
+				"-DCMAKE_BUILD_TYPE=RelWithDebugInfo"
+			],
+			"sources": [
+				{
+					"type": "git",
+					"url": "https://github.com/matwey/fips3",
+					"branch": "master"
+				},
+				{
+					"type": "file",
+					"path": "space.fips.Fips.appdata.xml"
+				}
+			],
+			"post-install": [
+				"install -D -m 644 space.fips.Fips.appdata.xml /app/share/appdata/space.fips.Fips.appdata.xml"
+			],
+			"test-rule": "test",
+			"run-tests": true
```

Comment:
Build fails because of this: https://flathub.org/builds/#/builders/42/builds/301

Seemingly you can't have this in a sandboxed build, which is what flathub uses.

Reply:
What is wrong with `test-args` in sandboxed build? Cmake generates `make test` whereas Flatpak wants to run `make check`. I think that this is just command tuning argument.

Reply:
Yeah, that's a flatpak-builder bug.

Reply:
Not exactly a flaw in logic, I just don't get the point of limiting it. @TingPing @alexlarsson any ideas? `test` is rather popular choice to execute test suite so it makes no sense to restrict it (if at all but I guess it's for security reasons).

Reply:
Sure, not arguing sandboxing builds, but we already allow build-commands, so why limit make targets for build/tests?